lib/consoletest.pm

```diff
@@ -49,13 +49,19 @@ Method executed when run() finishes and the module has result => 'fail'
 
 =cut
 sub post_fail_hook {
-    my ($self) = shift;
-    select_console('log-console');
-    $self->SUPER::post_fail_hook;
-    $self->remount_tmp_if_ro;
-    $self->export_logs_basic;
-    # Export extra log after failure for further check gdm issue 1127317, also poo#45236 used for tracking action on Openqa
-    $self->export_logs_desktop;
+    if (!get_var('PUBLIC_CLOUD')) {
```

Comment:
maybe we can just remove `consoletest` as a base ? As far as I can tell reload of `post_fail_hook` and test_flags is only things which we getting by this . In case we don't need this we can just drop base class

Reply:
ignore comment above , I get your point **why** you actually need this . Just maybe you need to add some comment here why you doing this so there will be less questions like one above ? :)

Reply:
My suggestion is:
```suggestion
    # On Public Cloud everything is done via SSH so the code below doesn't work
    # TODO: Make some reasonable base post_fail_hook for Public Cloud
    if (!get_var('PUBLIC_CLOUD')) {
```